definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType.json

```diff
@@ -57,7 +57,7 @@
     "LiveFrom": "16/05/2019",
     "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
If you do this, you'll change prod behaviour. What you want is to copy this (as it is in prod) to the "prod" file and add your version to your feature-toggled file.

Reply:
Youre correct @jvictorcf . Ive made my updates accordingly
Here: Remove "create" from core file and move case event to `-prod` and `-nonprod` files: aeea6bf938d867526b11eb09087fbff29bf530c2
and here: _Sets case type back to original_ - definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType.json